extensions/wikia/AdEngine/js/template/porvata.js

```diff
@@ -148,6 +157,13 @@ define('ext.wikia.adEngine.template.porvata', [
 
 		log(['show', params], log.levels.debug, logGroup);
 
+		if (!isVideoAutoplaySupported()) {
```

Comment:
What do you think about moving it below `loadVeles` call? It will mark bid as used and it will prevent second try of loading Porvata in `INCONTENT_LEADERBOARD`